### DISCUSSION TITLE
plugin: Idea Plugin

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [dart-lightning/lndart.cln#34] [Ideas]
We started to think about some ideas about the demo plugin that we need to build, I have some ideas like the following one:

- A doctor plugin that analyzes the failure on core lightning, like why a payment is failing (issue https://discord.com/channels/966807868777054208/967153864686076044/974028961623470101)
- payment proxy, a payment plugin that help use different type of payment that cln provide in a easy way for the end user

**Comment by swaptr** [comment]
Concept ACK.

**Comment by vincenzopalazzo** [comment]
Another idea https://github.com/ElementsProject/lightning/issues/5320#issuecomment-1172925564
